asf_search/export/csv.py

```diff
@@ -20,32 +22,150 @@
     ('sizeMB', ['DataGranule', 'ArchiveAndDistributionInformation', 0, 'Size']),
     ('insarStackSize', ['AdditionalAttributes', ('Name', 'INSAR_STACK_SIZE'), 'Values', 0]),
 ]
+
+fieldnames = (
+"Granule Name",
+"Platform",
+"Sensor",
+"Beam Mode",
+"Beam Mode Description",
+"Orbit",
+"Path Number",
+"Frame Number",
+"Acquisition Date",
+"Processing Date",
+"Processing Level",
+"Start Time",
+"End Time",
+"Center Lat",
+"Center Lon",
+"Near Start Lat",
+"Near Start Lon",
+"Far Start Lat",
+"Far Start Lon",
+"Near End Lat",
+"Near End Lon",
```

Comment:
So minor, but what do you think about everything inside the tuple be indented out for readability?